backend/route/contributors.route.js

```diff
@@ -0,0 +1,32 @@
+const Boom = require('@hapi/boom');
+const axios = require('axios');
+const linkedIn = require('../shared/contributors')
+
+const getLinkedInUrl = username => {
+    return linkedIn[username]
+}
+
+// Get all contributors
+module.exports = (server) => {
+    server.route({
+        method: 'GET',
+        path: '/api/v1/contributors',
+        handler: async (request, h) => {
+            try {
+                const { data } = await axios.get(
+                    `https://api.github.com/repos/MintbeanHackathons/MintbeanPlatform/contributors`
+                );
+                const contributors = data;
+                for (const contributor of contributors) {
+                    const { data } = await axios.get(contributor.url);
+                    contributor.name = data.name;
+                    contributor.portfolio = data.blog;
```

Comment:
In the portfolio link we have to ensure that the link begins with `http://` because otherwise the link it is going to be taken as an url within the mintbean.io domain, that's why the porfolio links are pointing to a page that does not exist within the mintbean.io domain and not to the actual portfolio.

We can do this on the frontend as well but I definitely prefer to do it on this line.

I suggest to create a function
```suggestion
                    contributor.portfolio = prefixHttp(data.blog);
```
And the definition of this function check if the url is an empty string so do nothing, as well as if begins with http://, if not prefix http://. that should be enough to solve this broken url

Reply:
I made this change, but I'm not sure it's required. And I'm betting it may add more headaches than it solves. @monarchwadia, do you have an opinion here?

Reply:
I added a function that adds `http://` if it isn't already there.